config/config.yaml

```diff
@@ -52,9 +65,14 @@ diffexp:
     volcano-plot: 0.05
     ma-plot: 0.05
     qq-plot: 0.05
+  # Optional (comment in to use): provide a list of genes that shall be shown in a heatmap
+  # and for which bootstrap plots (see below) shall be created.  
+  genes_of_interest:
+    activate: false
+    genelist: "resources/gene_list.tsv"
```

Comment:
```suggestion
  # genes_of_interest:
  #   activate: false
  #   genelist: "resources/gene_list.tsv"
```

Reply:
Not a good idea, I withdraw this.

Reply:
Can you please make genes_of_interest with its subitems a required element in the config schema?

Reply:
Noted. Added.

Reply:
Noted. Changed